backend/src/utils/consolidateQueryResult.ts

```diff
@@ -50,24 +50,43 @@ function deduplicateEntities<T extends HasId>(
  * @param queryResult The query-result to filter
  */
 export default function consolidateQueryResult(
-  queryResult: QueryResult
+  queryResult: QueryResult,
+  includeSubsidiary = false
 ): QueryResult {
   const { nodes, edges } = queryResult;
   const dedupNodes = deduplicateEntities(nodes, false);
```

Comment:
FYI: Duplicated Entities and Nodes might™ be fixed now. Open Issue #165 to validate that.